### PR TITLE
feat(core): add version field to meta 🙀

### PR DESCRIPTION
**Pull request by mcdurdin**
Also adds semver package for validating the version field.

@keymanapp-test-bot skip

### CODE REVIEW

File: common/tools/hextobin/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "commander": "^3.0.0"
   },
-  "main": "build/hextobin.js",
+  "main": "build/index.js",
```
Comment (mcdurdin) [inline common/tools/hextobin/package.json:13]: Mistake from previous PR...

File: core/src/kmx/kmx_plus.h [hunk 2]
```diff
@@ -34,6 +34,8 @@ namespace kmx {
 #pragma warning ( disable : 4200 )
 #endif
 
+typedef KMX_DWORD KMXPLUS_STR;
```
Comment (mcdurdin) [inline core/src/kmx/kmx_plus.h:37]: Has no real meaning but helps to clarify that these are offsets to strings rather than plain DWORDs 😁

**Review by rc-swag — APPROVED** [review]
LGTM

File: core/src/kmx/kmx_plus.h (hunk 2 above)
Comment (srl295) [inline core/src/kmx/kmx_plus.h:37]:
```suggestion
/**
 * Indicates an offset into the strs table (0 = zero length)
 */
typedef KMX_DWORD KMXPLUS_STR;
```

**Review by srl295 — APPROVED** [review]

**Pull request merged**